src/lib/public/board/data/officers.json

```diff
@@ -1,718 +1,368 @@
+      "F21": { "title": "Webmaster", "tier": 2 },
+      "S22": { "title": "Webmaster", "tier": 2 },
+      "F22": { "title": "Vice President/Webmaster", "tier": 1 }
+    }
   },
   {
     "fullName": "Kevin Dillon",
-    "positions": {
-      "S21": {
-        "title": "Algo Officer",
-        "tier": 15
-      }
-    },
-    "picture": "kevin-dillon.webp"
+    "picture": "kevin-dillon.webp",
+    "positions": { "S21": { "title": "Algo Officer", "tier": 12 } }
   },
   {
     "fullName": "Diamond Dinh",
-    "socials": {
-      "github": "diamondburned"
-    },
+    "picture": "diamond-dinh.webp",
+    "socials": { "github": "diamondburned" },
     "positions": {
-      "S22": {
-        "title": "Dev Project Manager",
-        "tier": 22
-      },
-      "F22": {
-        "title": "Special Events Officer",
-        "tier": 25
-      }
-    },
-    "picture": "diamond-dinh.webp"
+      "S22": { "title": "Dev Project Manager", "tier": 18 },
+      "F22": { "title": "Special Events Officer", "tier": 7 }
+    }
   },
   {
     "fullName": "Eduardo Gomez",
-    "positions": {
-      "S21": {
-        "title": "nodebuds Officer",
-        "tier": 23
-      }
-    },
-    "picture": "eduardo-gomez.webp"
+    "picture": "eduardo-gomez.webp",
+    "positions": { "S21": { "title": "nodebuds Officer", "tier": 22 } }
   },
   {
     "fullName": "Nurhaliza Hassan",
+    "picture": "nurhaliza-hassan.webp",
     "positions": {
-      "S22": {
-        "title": "Marketing Director",
-        "tier": 12
-      },
-      "F22": {
-        "title": "Marketing Co-Lead",
-        "tier": 23
-      }
-    },
-    "picture": "nurhaliza-hassan.webp"
+      "S22": { "title": "Marketing Director", "tier": 8 },
+      "F22": { "title": "Marketing Co-Lead", "tier": 8 }
+    }
   },
   {
     "fullName": "Lisa Hong",
-    "positions": {
-      "S21": {
-        "title": "Create Officer",
-        "tier": 17
-      }
-    },
-    "picture": "lisa-hong.webp"
+    "picture": "lisa-hong.webp",
+    "positions": { "S21": { "title": "Create Officer", "tier": 16 } }
   },
   {
     "fullName": "Joshua Hughes",
-    "positions": {
-      "S21": {
-        "title": "ICC Representative",
-        "tier": 8
-      }
-    },
-    "picture": "joshua-hughes.webp"
+    "picture": "joshua-hughes.webp",
+    "positions": { "S21": { "title": "ICC Representative", "tier": 23 } }
   },
   {
     "fullName": "Iftekharul Islam",
-    "positions": {
-      "S22": {
-        "title": "Algo Officer",
-        "tier": 15
-      }
-    },
-    "picture": "iftekharul-islam.webp"
+    "picture": "iftekharul-islam.webp",
+    "positions": { "S22": { "title": "Algo Officer", "tier": 12 } }
   },
   {
     "fullName": "Ibrahim Israr",
-    "positions": {
-      "S22": {
-        "title": "Secretary",
-        "tier": 3
-      }
-    },
-    "picture": "ibrahim-israr.webp"
+    "picture": "ibrahim-israr.webp",
+    "positions": { "S22": { "title": "Secretary", "tier": 3 } }
```

Comment:
The updated "tier" for Secretary is '4'

` "Secretary": { "id": 4, "index": 300 },`